src/config/language.c

```diff
@@ -53,6 +74,33 @@ static struct str_map {
     u16 pos;
 } lookupmap[MAX_STRINGS];
 
+static const char* LoadString(u16 pos, const char *str)
+{
+#if !SUPPORT_DYNAMIC_LOCSTR
+    (void)str;
```

Comment:
use `if(! SUPPORT_DYNAMIC_LOCSTR)` here

Reply:
not possible here as strings variable is not defined.